doc/whats_new.rst

```diff
@@ -51,6 +51,11 @@ Decomposition, manifold learning and clustering
   division on Python 2 versions. :issue:`9492` by
   :user:`James Bourbeau <jrbourbeau>`.
 
+- In :class:`decomposition.pca` selecting a n_components parameter greater than
```

Comment:
I would either phrase both changes in terms of what the problem was or what the fix was. Right now you you describe the error for the first change and the fix for the second.